torch/csrc/jit/script/init.cpp

```diff
@@ -5,6 +5,48 @@ namespace torch {
 namespace jit {
 namespace script {
 
+using ResolutionCallback = std::function<py::function(Graph*, std::string)>;
+
+struct PythonResolver : public Resolver {
+  PythonResolver(ResolutionCallback rcb) : rcb(rcb) {}
+
+  std::vector<Value*> resolveCall(SourceRange location, Node* n) override {
+    AutoGIL ag;
+    py::function func;
+    try {
+      func = rcb(n->owningGraph(), n->kind().toString());
```

Comment:
Let's just have the python function return `None` rather than throw. That case already needs to be caught below.